docs/gcloud-auth.rst

```diff
@@ -58,15 +58,68 @@ properties of your local environment in the following order
 #. Credentials provided by the Google Cloud SDK (via ``gcloud auth login``)
 #. Application running in Google Compute Engine
 
-Loading Credentials Explicitly
-------------------------------
+Explicit Credentials
+====================
 
-In addition, the
+The Application Default Credentials discussed above can be useful if your
+code needs to run in many different environments or if you just don't
+want authentication to be a focus in your code.
+
+However, if your code will only run in one place, you may want to
```

Comment:
Maybe an example here? (If you have a script that always needs to run as a specific account in many different environments?)